day-to-day.md

```diff
@@ -4,51 +4,47 @@
 
-Our benchmark number is between 32 and 40 tracked hours, and we operate in good
-faith that we are doing the best we can for the company without burning out.
-
-> Question: at what point do we, as a company, start pestering people about
-> their hours? Should one make up the time spent in a long lunch that wasn't a
-> meeting?
+In order for us to fullfil our obligations to clients and to best work with your team, we ask that you are present at predictable times. Our benchmark number is between 35 and 40 tracked hours, and we operate in good faith that we are doing the best we can for the company without burning out. However, if you are not meeting expectations for reliable time tracking or attendance, another teammate will reach out to you to ask if the issues impeding reliability are fixable. We don't expect you to be logged in from 9-5, but appropriate and reliable communication is paramount when working with a team and for clients.
 
 - Not sure what to track to? Ask! (Seriously.)
 - We round up or down to the nearest quarter hour to streamline our billing process
 - Everyone can see everyone's time.
-  - Don't panic, no one's stalking you. Except maybe Tony.
+  - Don't panic, no one's stalking you (see code of conduct). Except maybe Tony.
   - Feel free to check out other people's time sheets if you're curious how
     they're tracking things.
 - Be honest in tracked time, even if it's short of 40 hours
 - If time is not tied to a task number add a description of what you did (future
   you will be pleased)
-- When you're not doing client work, track to Internal
+- When you're not doing client work, track to _Internal_
   - There are lots of buckets in Internal, check them out
   - It gives you (and the team) insight into your past work
+- Include details for client work. You do not have to go overboard, but if you are not tracking to a specific development task, try to explain what you spent your time doing.
 
 Try to have the day's hours in and rounded to the quarter hour by the end of the day. If we feel like we have a lot of billable work but aren't tracking a high number of billable hours, that
 will likely trigger a conversation around team productivity.
 
 ### Flex time
 
-Clear time tracking allows for a flexible work schedule without feeling guilty. This allows you to mold your work hours to better fit your personal schedule. For example, if you need to get a haircut, and the best time to schedule that is 2pm on a Thursday, you can make up the hour you miss on
-another day (prior or subsequent).
+A clear system for time tracking allows for a flexible work schedule without feeling guilty. This allows you to mold your work hours to better fit your personal schedule. For example, if you need to get a haircut, and the best time to schedule that is 2pm on a Thursday, you can make up the hour you miss on
+another day (prior or subsequent). Communicating your away from keyboard time to your teammates is a nice thing to do.
```

Comment:
💭 Possible reword. Doesn't have to be this exact wording, but I think "Communicating your away from keyboard time" can be improved.

```suggestion
another day (prior or subsequent). We appreciate you giving teammates a heads up when you'll be out of pocket.
```

Reply:
I pushed this a little more, let me know what you think.